include/DhyanaCamera.h

```diff
@@ -70,6 +70,40 @@ class LIBDHYANA_API Camera
         Ready, Exposure, Readout, Latency, Fault
     } ;
 
+    enum TucamTriggerMode
+    {
+      TriggerStandard = TUCCM_TRIGGER_STANDARD,
+      TriggerSynchronous = TUCCM_TRIGGER_SYNCHRONOUS,
+      TriggerGlobal = TUCCM_TRIGGER_GLOBAL,
+      //TriggerSoftware = TUCCM_TRIGGER_SOFTWARE, do not map, this mode is used for Lima IntTrigSingle and Timer to retrig
+    };
+
+    enum TucamTriggerEdge
+    {
+      EdgeRising = TUCTD_RISING,
+      EdgeFalling = TUCTD_FAILING,
+    };
+
+    enum TucamSignal
+    {
+      SignalTrigIn = TUOPT_IN,        //copy of the trigger IN
+      SignalStart = TUOPT_EXPSTART,   // Exposure start (rolling)
+      SignalGlobal = TUOPT_EXPGLOBAL, // Global exposure
+      SignalReadEnd = TUOPT_READEND   // readout end
+    };
+    enum TucamSignalEdge
+    {
+      SignalEdgeRising = TUOPT_RISING,
+      SignalEdgeFalling = TUOPT_FAILING,
+    };
+
+    enum TucamGain
+    {
+      GainHDR  = TUGAIN_HDR,
+      GainHigh = TUGAIN_HIGH,
+      GainLow  = TUGAIN_LOW
```

Comment:
could you please rename the enum keys by adding a small k eg: 
enum TucamSignalEdge
    {
      kSignalEdgeRising = TUOPT_RISING,
      kSignalEdgeFalling = TUOPT_FAILING,
    };